corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap3/sub_group.html

```diff
@@ -41,7 +41,8 @@
                 "></span>
           <button class="btn btn-danger del pull-right" href="#" aria-label="{% trans_html_attr 'Delete Repeat' %}"
                   data-bind="
-                      visible: isRepetition,
+                      visible: showDelete,
+                      enabled: hasNoPendingAnswer(),
```

Comment:
I just noticed there's another delete button a few lines down, for collapsible repeats - should this change also be made there?